Intersection/google-maps-clustering
# App 788 add access to markers

How to test
1) Add a working Google API Key to strings.xml of sample app as google_maps_key
2) Run the sample app
3) Zoom in to see normal pins
4) Zoom in further to see rotated pins
5) Look for any choppy transitions from clusters to pins
